src/@utils/ddo.ts

```diff
@@ -2,14 +2,21 @@ import { Asset, DDO, Service } from '@oceanprotocol/lib'
 
 export function getServiceByName(
   ddo: Asset | DDO,
-  name: 'access' | 'compute'
+  name: 'access' | 'compute' | 'metadata'
```

Comment:
There is no metadata service in v4 anymore, so this should stay as:

```suggestion
  name: 'access' | 'compute'
```